docs/account_management/gitlab_account.md

```diff
@@ -64,3 +64,108 @@ If you are a collaborator with a XIAS account you'll need to follow a different
 
     XIAS account researchers can only be granted access if their sponsor adds the GitLab URL to the list of approved URIs. Please see [XIAS Sites](./xias/pi_site_management.md) for more information.
 <!-- markdownlint-enable MD046 -->
+
+## UAB GitLab Roles and User Permissions
+
+Managing roles and user permissions within GitLab is essential for maintaining control over repositories and collaboration. Within UAB Research Computing's GitLab instance, a user (Owner) can assign and update user roles in a group. There are several roles for users within GitLab, and these roles determine the level of permissions a user will have within a Group or project. The available default roles within a project are;
+
+1. Guest
```

Comment:
These are also listed in the table. You could remove this list and reference the table column names, if you wanted the article to be more compact.